client/src/components/CreatePost.tsx

```diff
@@ -37,144 +54,185 @@ function CreatePost() {
-                data-testid="bodyTextField"
-                placeholder="description"
-                multiline
-                rows={6}
-                onChange={(e) => setForm({ ...form, body: e.target.value })}
-              />
-            </Grid>
-
-            <Grid item xs={12} md={4}>
-              <TextField
-                required
-                label="Upload Thumbnail"
-                type="file"
-                fullWidth
-                data-testid="fileField"
-                InputLabelProps={{ shrink: true }}
-                size="small"
-                onChange={(e) => handleImageUpload(e)}
-              />
-            </Grid>
-
-            <Grid item xs={12} md={3}>
-              <TextField
-                label="Event Capacity"
-                placeholder="40"
-                fullWidth
-                size="small"
-                onChange={(e) =>
-                  setForm({ ...form, eventCapacity: e.target.value })
-                }
-              />
-            </Grid>
-
-            <Grid item xs={12} md={5}>
-              <TextField
-                fullWidth
-                label="Location"
-                placeholder="Deerfield Hall"
-                size="small"
-                onChange={(e) =>
-                  setForm({ ...form, eventLocation: e.target.value })
-                }
-              />
+        New Post
+      </Button>
+      <Dialog fullScreen open={isOpen} onClose={closeDialog}>
+        <Container component="main" maxWidth="md">
+          <Box
+            sx={{
+              marginTop: 8,
+              display: "flex",
+              flexDirection: "column",
+              alignItems: "center",
+            }}
+          >
+            <Grid container>
+              <Grid item xs={4}>
+                <Button
+                  onClick={closeDialog}
+                  size="large"
+                  variant="contained"
+                  data-testid="backButton"
+                >
+                  <ArrowBack />
+                  Back
+                </Button>
+              </Grid>
             </Grid>
 
-            <Grid item xs={12}>
-              <TextField
-                fullWidth
-                label="Tags (Separate tags by a comma)"
-                placeholder="Clubs, Math, MCS"
-                size="small"
-                onChange={(e) => setForm({ ...form, tags: e.target.value })}
-              />
-            </Grid>
-          </Grid>
-
-          {/* form  done */}
-          <Box sx={{ mt: 2 }}>
-            <Divider />
-          </Box>
-
-          <Grid container spacing={2} justifyContent="center">
-            <Grid item xs={5} md={3}>
-              <Tooltip title="Enter all required fields!">
-                <Box>
+            <Typography
+              component="h3"
+              variant="h3"
+              fontWeight="bold"
+              paddingTop="10px"
+            >
+              Create Post
+            </Typography>
```

Comment:
Can this not just be an `<h3>...</h3>`?

Reply:
I think its fine right? Like in the case we might want to apply a theme to this? Unless theming still works with plain html tags, I'm not too sure.

Reply:
It does. Two options 1/ useTheme and apply via the style attribute 2/ use styled-components. It's an extra dependency and import that isn't *really* necessary. 

It's OK to leave it in, I'm not going to be picky about it, but I would just use a regular HTML tag.